packages/server/src/models/TokenModel.ts

```diff
@@ -2,10 +2,11 @@ import { Token, User, Uuid } from '../services/database/types';
 import { ErrorForbidden, ErrorNotFound } from '../utils/errors';
 import { uuidgen } from '@joplin/lib/uuid';
 import BaseModel from './BaseModel';
+import { Day } from '../utils/time';
 
 export default class TokenModel extends BaseModel<Token> {
 
-	private tokenTtl_: number = 7 * 24 * 60 * 60 * 1000;
+	private tokenTtl_: number = 7 * Day;
```

Comment:
Please remove this - I agree it's ok as a change but I prefer to keep commits focused on one specific issue.

Reply:
Done.